ios/browser/api/brave_wallet/brave_wallet_provider_delegate_ios.mm

```diff
@@ -42,6 +43,7 @@
 }
 
 void BraveWalletProviderDelegateBridge::GetAllowedAccounts(
+    mojom::CoinType type,
```

Comment:
We're missing C++ implementations for the new methods

Reply:
fixed iin 09c8e944d6359048b35a8cb55113e07ed5b2163a